admin/tests/integration/components/users/user-detail-personal-information_test.js

```diff
@@ -178,58 +176,17 @@ module('Integration | Component | users | user-detail-personal-information', fun
         const screen = await render(
           hbs`<Users::UserDetailPersonalInformation @user={{this.user}} @removeAuthenticationMethod={{this.removeAuthenticationMethod}}/>`
         );
-        await clickByName('Supprimer');
+        await click(screen.getAllByRole('button', { name: 'Supprimer' })[0]);
 
         // when
-        await clickByName('Oui, je supprime');
+        await click(screen.getByRole('button', { name: 'Oui, je supprime' }));
 
         // then
         assert.ok(removeAuthenticationMethodStub.called);
         assert.dom(screen.queryByRole('heading', { name: 'Merci de confirmer' })).doesNotExist();
         assert.dom(screen.queryByRole('button', { name: 'Annuler' })).doesNotExist();
         assert.dom(screen.queryByRole('button', { name: 'Confirmer' })).doesNotExist();
       });
```

Comment:
Question : pourquoi avoir supprimer ce test  `should display an error message when the admin member try to remove the last authentication method`alors que le code `admin/app/components/users/user-detail-personal-information.js` qui génère cette erreur n'a pas été supprimé

Reply:
J'ai oublié de supprimer aussi le code.

En faite il y a dans le composant authentication-method une vérification du nombre de méthodes de connexions permettant l'affichage ou non des boutons Supprimer. Vu que cette erreur n'est disponible que si l'on tente de supprimer la dernière méthode de connexion, j'ai trouvé impossible que ce cas arrive.

Quand on regarde le cas du test, il n'est pas logique. On ajoute 2 méthodes de connexions puis on en supprime 1 seule et on stub pour forcer l'erreur.

Reply:
Je proposerais plutôt de remanier ce test qui vérifie toujours qu'**un admin ne peut pas supprimer une méthode de connexion quand l'utilisateur n'a plus qu'un seule méthode**. C'est un élément métier fort.
C'est impossible actuellement mais si on veut refactorer plus tard, on est plus serein car on saura si le test est toujours vert, qu'on a rien cassé.

Reply:
True, je rajoute ce test manquant.